Client/Client.Qt/Widgets/TextEdit.cpp

```diff
@@ -96,33 +96,34 @@ void TextEdit::styleButtonClick(const QString& symbolStart, const QString& symbo
     }
 }
 
-void TextEdit::delSymbolsInSelection(QString& text, int& start, int& end, int symbolSize)
+void TextEdit::delSymbolsInSelection(QString& text, int start, int end, int symbolSize) const
```

Comment:
Explain next time what happend here?

Reply:
Put response here, please

Reply:
@Orange-hanter , @ivanboika . We discussed this at the meeting. You closed this issue.